project/src/test/java/com/google/sps/EventDetailTest.java

```diff
@@ -4,6 +4,7 @@
 import com.google.sps.data.Event;
 import com.google.sps.data.User;
 import com.google.sps.servlets.EventDetailServlet;
+import com.google.sps.servlets.FilteredEventServlet;
```

Comment:
I think you should separate this into 2 tests for clarity.

Reply:
done